### PR TITLE
Create a dedicated vcpkg toolchain

### PR DESCRIPTION
**Pull request by m-kuhn**
Encapsulates nuget logic and overlay logic inside cmake

Credits to @Hoikas

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,6 +18,8 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 
 include(GetGitRevisionDescription)
 
+include(VcpkgToolchain)
+
 set(VCPKG_APPLOCAL_DEPS ON)
 set(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
```
Comment (m-kuhn) [inline CMakeLists.txt:24]:
```suggestion
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 8b4a0333fe87bd4bc349dda22c985986951c67aa:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-bfe0d8eba4262647031bddba5e91ac1d4ee5affc-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-bfe0d8eba4262647031bddba5e91ac1d4ee5affc-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-bfe0d8eba4262647031bddba5e91ac1d4ee5affc-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-bfe0d8eba4262647031bddba5e91ac1d4ee5affc-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9a435c4be055797799d1868ea5e15b539fde1d82:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-734310b2b36bde62f51662ac6ec441caa0fa72f9-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-734310b2b36bde62f51662ac6ec441caa0fa72f9-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-734310b2b36bde62f51662ac6ec441caa0fa72f9-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-734310b2b36bde62f51662ac6ec441caa0fa72f9-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4cdc3942b9091d094c39ef476a876ea32140cb05:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4cdc3942b9091d094c39ef476a876ea32140cb05:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-35ae8d30fc9e9e77bbba353f01252af3b827ff96-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 6a480b3be333f70302e97a34fd27bf8a3cd220cb:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-19ad35096581ace3420da652ea83f2d9d2b124e4-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-19ad35096581ace3420da652ea83f2d9d2b124e4-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-19ad35096581ace3420da652ea83f2d9d2b124e4-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-19ad35096581ace3420da652ea83f2d9d2b124e4-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a90c64523807d3c7e12943e0b60208e36196d18f:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-adf4d79a71c84403831e022544a9b717a7c19e8d-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-adf4d79a71c84403831e022544a9b717a7c19e8d-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-adf4d79a71c84403831e022544a9b717a7c19e8d-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-adf4d79a71c84403831e022544a9b717a7c19e8d-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 242a701c630d0534c6536760f2abec7e6c98272a:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-552f6322d4c78b1f62b51ead9f7db8549fafa2e6-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-552f6322d4c78b1f62b51ead9f7db8549fafa2e6-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-552f6322d4c78b1f62b51ead9f7db8549fafa2e6-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-552f6322d4c78b1f62b51ead9f7db8549fafa2e6-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3be8bb8adbbc2a80f79def171c7f91b98a15851e:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-24b3953a9b715efd4240f1dfdc33cb7da3adaed6-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-24b3953a9b715efd4240f1dfdc33cb7da3adaed6-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-24b3953a9b715efd4240f1dfdc33cb7da3adaed6-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-24b3953a9b715efd4240f1dfdc33cb7da3adaed6-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3f42785e4ce26b4100755d409793e1d24015a9b4:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-f9fc139a66446ac294a648bd087707c8115ced4c-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-f9fc139a66446ac294a648bd087707c8115ced4c-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-f9fc139a66446ac294a648bd087707c8115ced4c-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-f9fc139a66446ac294a648bd087707c8115ced4c-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 283700d376f61caf454ec9692fd6e016c9dac98e:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-cf3a4567c20a3fa3b407dfbc5ea974fea22bc433-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-cf3a4567c20a3fa3b407dfbc5ea974fea22bc433-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-cf3a4567c20a3fa3b407dfbc5ea974fea22bc433-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-cf3a4567c20a3fa3b407dfbc5ea974fea22bc433-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 6f78ed10cfc1704af0e3757505573c7257f26904:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-2863f7778f11b2fd9f01dcc9e5c253f6590afe2f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-2863f7778f11b2fd9f01dcc9e5c253f6590afe2f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-2863f7778f11b2fd9f01dcc9e5c253f6590afe2f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-2863f7778f11b2fd9f01dcc9e5c253f6590afe2f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for fd579b05ffdb69f79c6ff27e6f2ce9b5fe914849:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-e8f70ca7741e8f9ecad086e598414049ddc81452-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-e8f70ca7741e8f9ecad086e598414049ddc81452-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-e8f70ca7741e8f9ecad086e598414049ddc81452-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-e8f70ca7741e8f9ecad086e598414049ddc81452-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 78a4649b05222bf3de951c97462231aa63f1da72:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-3b3095e96e4612c3f0522aca483ca70cae5c5193-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-3b3095e96e4612c3f0522aca483ca70cae5c5193-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-3b3095e96e4612c3f0522aca483ca70cae5c5193-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-3b3095e96e4612c3f0522aca483ca70cae5c5193-x86.apk)

File: .github/workflows/vcpkg.yml [hunk 2]
```diff
@@ -207,6 +207,7 @@ jobs:
           name: build-logs-${{ matrix.triplet }}
           path: |
             ${{ env.CMAKE_BUILD_DIR }}/vcpkg-manifest-install.log
+            ${{ github.workspace }}/vcpkg/base/buildtrees/**/*.log
```
Comment (m-kuhn) [inline .github/workflows/vcpkg.yml:210]: Thanks @suricactus

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3ca7a4588f7ed5e89d80a554103c31aa08d19564:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a19436f834fc6ef44c07666b4f854de05a7248ae-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a19436f834fc6ef44c07666b4f854de05a7248ae-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a19436f834fc6ef44c07666b4f854de05a7248ae-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a19436f834fc6ef44c07666b4f854de05a7248ae-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a35b50dac981b36842c5315cb6e89781166cddf3:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-0a2063f0f9c873cd2375bf1e31fbc9fd0f239f91-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-0a2063f0f9c873cd2375bf1e31fbc9fd0f239f91-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-0a2063f0f9c873cd2375bf1e31fbc9fd0f239f91-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-0a2063f0f9c873cd2375bf1e31fbc9fd0f239f91-x86.apk)

File: .github/workflows/vcpkg.yml (hunk 2 above)
Comment (suricactus) [reply .github/workflows/vcpkg.yml:233]: No idea how I helped, but you are welcome :)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 11bf30fe25a2007dc2c63d227f22a90032d991a6:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-59b5a1ec9aefa64312e1a55ea1087d1010f85434-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-59b5a1ec9aefa64312e1a55ea1087d1010f85434-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-59b5a1ec9aefa64312e1a55ea1087d1010f85434-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-59b5a1ec9aefa64312e1a55ea1087d1010f85434-x86.apk)

File: cmake/VcpkgToolchain.cmake [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  #    RESULT_VARIABLE _RESULT)
+  #  if(_RESULT EQUAL 0)
+  #    message(STATUS "Setup nuget source [github] - done")
+  #  else()
+  #    message(STATUS "Setup nuget source [github] - failed")
+  #    message(STATUS "Output:")
+  #    message(STATUS ${_OUTPUT})
+  #    message(STATUS "Error:")
+  #    message(STATUS ${_ERROR})
+  #  endif()
```
Comment (m-kuhn) [inline cmake/VcpkgToolchain.cmake:85]: Replaced by configure_file
```suggestion
```

File: .github/workflows/vcpkg.yml (hunk 2 above)
Comment (m-kuhn) [reply .github/workflows/vcpkg.yml:234]: You mentioned that there are warnings on those builds when it tries to upload logs

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9125736c33f867080e6c8c84e7f83ff129a380fe:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-545257aeefdbbaf38b55e7b20e6a1f2dc35b16a7-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-545257aeefdbbaf38b55e7b20e6a1f2dc35b16a7-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-545257aeefdbbaf38b55e7b20e6a1f2dc35b16a7-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-545257aeefdbbaf38b55e7b20e6a1f2dc35b16a7-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d900fdd17669b6299f04f5e6b41329a9d5939bb7:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-69cb4b9b0ac098595068227a2c5e196c77cfce9e-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-69cb4b9b0ac098595068227a2c5e196c77cfce9e-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-69cb4b9b0ac098595068227a2c5e196c77cfce9e-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-69cb4b9b0ac098595068227a2c5e196c77cfce9e-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for b575b096b677190eac6ba20db8ec1341d71ea81b:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-375ba8f739240a1e03d17ed5631b9fda14ba50e0-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-375ba8f739240a1e03d17ed5631b9fda14ba50e0-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-375ba8f739240a1e03d17ed5631b9fda14ba50e0-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-375ba8f739240a1e03d17ed5631b9fda14ba50e0-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for caa3941d1f6b4c67d1700b1348199818f007f1a0:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-b11f5e043552441952d6efdf502ff4147dafe322-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-b11f5e043552441952d6efdf502ff4147dafe322-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-b11f5e043552441952d6efdf502ff4147dafe322-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-b11f5e043552441952d6efdf502ff4147dafe322-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 518616bd159ef19a14e9261bb170f4a85a60c283:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-5b50c68ee7a84234300e283613fdd607ea746036-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-5b50c68ee7a84234300e283613fdd607ea746036-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-5b50c68ee7a84234300e283613fdd607ea746036-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-5b50c68ee7a84234300e283613fdd607ea746036-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0c9c9e49ae507675a19fdb31ca2afcf827d276fa:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a10d9924331da9425c7b827fa03771fe4a889dff-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a10d9924331da9425c7b827fa03771fe4a889dff-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a10d9924331da9425c7b827fa03771fe4a889dff-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-a10d9924331da9425c7b827fa03771fe4a889dff-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a2eb1ed30da9394bfcfe99c8b3859da12dd41d09:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-8114d84911c0f7b93e2926b3d9142208d4f7384e-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-8114d84911c0f7b93e2926b3d9142208d4f7384e-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-8114d84911c0f7b93e2926b3d9142208d4f7384e-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2329-8114d84911c0f7b93e2926b3d9142208d4f7384e-x86.apk)

**Review by nirvn — APPROVED** [review]

**Pull request merged**